<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window01</class>
 <widget class="QMainWindow" name="Window01">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window01</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <item row="0" column="0">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton1">
      <property name="text">
       <string>选择1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton2">
      <property name="text">
       <string>选择2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton3">
      <property name="text">
       <string>选择3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton4">
      <property name="text">
       <string>选择4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
